### PR TITLE
docs: readmes fixed according to review

### PR DESCRIPTION
**Pull request by allaVolkov**
Before submitting a pull request, please ensure the following:

## Description

Please explain the changes you made.

Add a link to the design (if applicable).

### Checklist

- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Linting run locally successfully
- [ ] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,36 +6,37 @@
 [![DependentBot](https://api.dependabot.com/badges/status?host=github&repo=SAP/task-explorer)](https://dependabot.com/)
 [![REUSE status](https://api.reuse.software/badge/github.com/SAP/task-explorer)](https://api.reuse.software/info/github.com/SAP/task-explorer)
 
-# VSCode Task Explorer
```
Comment (sapirpol) [inline README.md:9]: Please change it to Task Explorer, because the name has changed

Comment (sapirpol) [inline README.md:9]: Please change it to Task Explorer, because the name has changed

File: README.md [hunk 2]
```diff
@@ -6,36 +6,37 @@
...
+# VS Code Task Explorer
```
Comment (sapirpol) [inline README.md:9]: Please change it to Task Explorer, because the name has changed

**Review by sapirpol — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (allaVolkov) [reply README.md]: done

**Review by sapirpol — APPROVED** [review]

**Pull request merged**